R/scale_colour_2dii.R

```diff
@@ -31,8 +31,19 @@
 #' ggplot(sda %>% filter(emission_factor_metric == "projected")) +
 #'  geom_line(aes(x = year, y = emission_factor_value, colour = sector)) +
 #'  scale_colour_2dii(palette = "1in1000", colour_groups = sda$sector)
+#'
+#'  ggplot(
+#'  market_share %>%
+#'    filter(
+#'    sector == "automotive",
+#'    year %in% c(2020, 2025),
+#'    metric == "projected")) +
```

Comment:
I find it confusing when the nesting and piping styles are mixed. I would do either one or the other.

So either

```r
data %>% filter(...) %>% ggplot(...)  + ...
```

or

```r
ggplot(filter(data, ...)) + ...
```